repository-hpi/src/main/java/com/nirima/jenkins/repo/project/ProjectsElement.java

```diff
@@ -47,10 +49,10 @@ public String getName() {
 
     public Collection<ProjectElement> getChildren() {
 
-        List<ProjectElement> elements = Lists.newArrayList(Iterators.transform(Jenkins.getInstance().getAllItems(BuildableItemWithBuildWrappers.class).iterator(),
-                new Function<BuildableItemWithBuildWrappers,ProjectElement>()
+        List<ProjectElement> elements = Lists.newArrayList(Iterators.transform(Jenkins.getInstance().getItems().iterator(),
```

Comment:
From a casual glance this looks like it would be moving in the opposite direction from what you need. `getAllItems` returns items even inside folders, whereas `getItems` returns only items at root level.